src/main/java/nz/ac/waikato/its/dspace/app/xmlui/aspect/featuredcontent/FeaturedContentController.java

```diff
@@ -0,0 +1,88 @@
+package nz.ac.waikato.its.dspace.app.xmlui.aspect.featuredcontent;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.content.Item;
+import org.dspace.core.ConfigurationManager;
+import org.dspace.core.Context;
+
+import java.io.*;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Created by jjung on 9/02/16.
+ */
+public class FeaturedContentController {
+
+    static HashMap<String,String> getFeaturedItem(){
+        HashMap<String,String> returnMap = null;
+
+        // configuration file location -> ${DSPACE_HOME}/config/modules/lconz-aspect.cfg
+        String featuredItemFile = ConfigurationManager.getProperty("lconz-aspect", "featured.content.file");
+        if (StringUtils.isEmpty(featuredItemFile)) {
+            return null;
+        }
+
+        Properties props = new Properties();
+        InputStream is = null;
+
+        try {
+            // file location -> ${DSPACE_HOME}/config/modules/featured-content
+            File propFile = new File(featuredItemFile);
+            if (!propFile.exists()) {
+                propFile.createNewFile();
+            }
+            props.load(new FileInputStream(propFile));
```

Comment:
Wouldn't it be better to use the `is` field you declared above so that the finally clause closes the right stream?